examples/graphql-authentication/.graphqlconfig.yml

```diff
@@ -1,17 +1,17 @@
 projects:
-  app: 
-    schemaPath: schema.graphql
+  app:
+    schemaPath: src/schema/schema.graphql
     extensions:
       endpoints:
-        default: 'http://localhost:4000'
-  db: 
-    schemaPath: generated-schema.graphql
+        default: http://localhost:4000
+  db:
+    schemaPath: src/generated/prisma.graphql
     extensions:
       endpoints:
-        default: 'http://localhost:4466/graphql-authentication'
+        default: http://localhost:4466/graphql-authentication
       prisma: prisma.yml
       codegen:
-        output: 
-          typings: generated-prisma.ts
+        output:
+          typings: src/generated/generated-prisma.ts
```

Comment:
let's also change the name of the generated TS files to just `prisma.ts` instead of `generated-prisma.ts`